android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerView.java

```diff
@@ -533,6 +535,7 @@ private void doInitializePlayer(boolean force) {
             exoPlayer.addAnalyticsListener(this);
             Player realPlayer = player.createForwardPlayer();
             exoDorisPlayerView.setPlayer(realPlayer);
+            exoDorisPlayerView.setAdPlayPauseEnabled(player instanceof ExoDorisYoSsaiPlayer);
```

Comment:
May change to `adType == AdType.YO_SSAI || adType == AdType.AMT_SSAI` ?

Reply:
resoleved